egs/sre16/v1/local/nnet3/xvector/tuning/run_xvector_1a.sh

```diff
@@ -127,7 +127,7 @@ fi
 dropout_schedule='0,0@0.20,0.1@0.50,0'
 srand=123
 if [ $stage -le 6 ]; then
-  steps/nnet3/train_raw_dnn.py --stage=$train_stage \
+  python steps/nnet3/train_raw_dnn.py --stage=$train_stage \
     --cmd="$train_cmd" \
     --trainer.optimization.proportional-shrink 10 \
     --trainer.optimization.momentum=0.5 \
```

Comment:
Is this change correct? I think the shebang at the top of the python file should take care of it.

Reply:
It works but not necessary.

Reply:
I think you should revert this change. It seems customary to rely on what's in the shebang.

Reply:
Sure.

Reply:
For example, look at https://github.com/kaldi-asr/kaldi/blob/master/egs/swbd/s5c/local/chain/tuning/run_tdnn_7m.sh line 173